app/Member.php

```diff
@@ -67,6 +73,14 @@ class Member extends Model
     public static $options_allowed_membership_type = array('1 year', 'Life');
     public static $options_membership_type_with_cost = array('1 year (£8)', 'Life (£15)');
 
+    public function setAllVisible()
+    {
+        $this->addVisible($this->getHidden());
+        $this->addVisible($this->guarded);
+        $this->addVisible($this->appends);
```

Comment:
Just wondering, out of curiosity: what do these lines do?
```
$this->addVisible($this->guarded);
$this->addVisible($this->appends);
```

Not that I think there's anything wrong with it per say (I trust you've done your homework and found it somewhere / needed this to make it work), but I don't understand what it does and what it's needed. From what I can find, `$this->appends` gives the accessors? But why `guarded`, since guarded/fillable is what can be used for mass assignment via `Member::create` instead of serialization?